src/utils/isCustomPropertySet.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Check whether a property is a custom properties
+ *
+ * @param {string} customPropertySet
+ * @return {boolean} If `true`, property is a custom properties
+ */
+export default function (customPropertySet) {
+  return (customPropertySet.slice(0, 2) === "--" && customPropertySet.slice(-1) === ":")
+}
```

Comment:
Agreed. I think `isCustomPropertySet` should be checking an entire rule node, rather than just its selector. 

Can we also check for a semicolon after the closing brace, as that is part of what defines a custom property set?